src/library/f3d_options.cxx

```diff
@@ -83,4 +84,72 @@ template double options::get<>(const std::string& name) const;
 template std::string options::get<>(const std::string& name) const;
 template std::vector<int> options::get<>(const std::string& name) const;
 template std::vector<double> options::get<>(const std::string& name) const;
+
+//----------------------------------------------------------------------------
+void options::initialize(options& options)
+{
+  // General
+  options.set("quiet", false);
+  options.set("verbose", false);
+
+  // Loader/Loading
+  options.set("animation-index", 0);
+  options.set("geometry-only", false);
+  options.set("progress", false);
+  options.set("camera-index", 0);
+  options.set("color", std::vector<double>({ 1., 1., 1. }));
```

Comment:
Would be awesome to support `std::initializer_list` to be able to call `options.set("color", { 1., 1., 1. });`

Reply:
indeed, outside of the scope though